modules/RTC/TraceablePeerConnection.js

```diff
@@ -2108,6 +2108,73 @@ TraceablePeerConnection.prototype._adjustLocalMediaDirection = function(
     return localDescription;
 };
 
+/**
+ * Converts an object to a config-like string.
+ *
+ * @param {Object} obj - The object to be converted into a config string.
+ * @returns {String} - The config string.
+ */
+TraceablePeerConnection.prototype.getConfigFromObject = function(obj) {
+    let config = '';
+
+    for (const key of Object.keys(obj)) {
+        config += `${key}=${obj[key]}; `;
+    }
+
+    return config.trim();
+};
+
+/**
+ * Munges the stereo flag as well as the opusMaxAverageBitrate in the SDP, based
+ * on values set through config.js, if present.
+ *
+ * @param {RTCSessionDescription} description that needs to be munged.
+ * @returns {RTCSessionDescription} the munged description.
+ */
+TraceablePeerConnection.prototype.mungeOpus = function(description) {
+    const parsedSdp = transform.parse(description.sdp);
+    const audio = parsedSdp.media.find(mLine => mLine.type === 'audio');
```

Comment:
Please note that this will not work for browsers that run Unified plan. On Firefox and Safari, there will be multiple m-lines for type audio, one for each remote audio source and all those m-lines need to be munged, not just the first one.